src/parser/warlock/demonology/modules/azerite/UmbralBlaze.js

```diff
@@ -27,13 +27,20 @@ class UmbralBlaze extends Analyzer {
     this.damage += event.amount + (event.absorbed || 0);
   }
 
+  get dps() {
```

Comment:
Looks like most of the modules calculate the dps the same way. Maybe this method could be pulled up to Analyzer or DPSAnalyzer extends Analyzer

Reply:
same for BoringSpellValueText